src/frontends/tensorflow/tests/convert_tricky_models.cpp

```diff
@@ -70,7 +70,7 @@ ov::OutputVector fake_translator_ragged_tensor_to_sparse(const ov::frontend::Nod
     add.get_tensor().add_names({node_name + ":1"});
     sub.get_tensor().add_names({node_name + ":2"});
 
-    return {mul, add, sub};
+    return {{"sparse_indices", mul}, {"sparse_values", add}, {"sparse_dense_shape", sub}};
```

Comment:
I think we should also allow using {"sparse_indices:0", mul}, {"sparse_values:0", add}, {"sparse_dense_shape:0", sub}`